### PR TITLE
analog: add commands for ultrasound water meter

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/analog/utils/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -1983,6 +2022,13 @@ CommandBinaryBuffer.prototype.getEventStatus = function ( hardwareType: number )
         status = bitSet.toObject(fourChannelBitMask, this.getUint16());
     } else if ( MTX_HARDWARE_TYPES.indexOf(hardwareType) !== -1 ) {
         status = bitSet.toObject(mtxBitMask, this.getUint16());
+    } else if ( hardwareType === hardwareTypes.US_WATER ) {
+        // ultrasound water meter
+        const event = bitSet.toObject(usWaterMeterEventBitMask, this.getUint8()) as unknown as IUSWaterMeterEvent;
+        status = {
```
Comment (DarkPark) [inline src/analog/utils/CommandBinaryBuffer.ts:2028]: maybe in one line?

File: src/analog/constants/hardwareTypes.ts [hunk 2]
```diff
@@ -124,3 +124,9 @@ export const IMP4AS = 15;
  * 1 channel
  */
 export const NBIOT = 24;
+
+/**
+ * RM installed inside ultrasound water meters.
+ * Based on  STM32WLE5.
```
Comment (DarkPark) [inline src/analog/constants/hardwareTypes.ts:130]:
```suggestion
 * Based on `STM32WLE5`.
```

File: src/analog/utils/CommandBinaryBuffer.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/analog/utils/CommandBinaryBuffer.ts:2028]: done

File: src/analog/constants/hardwareTypes.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/analog/constants/hardwareTypes.ts]: quote marks for all models added

**Review by DarkPark — APPROVED** [review]

**Pull request merged**